UV_excess_in_low_mass_stars/UV_excess_in_low_mass_stars.ipynb

```diff
@@ -0,0 +1,948 @@
+{
```

Comment:
<p>Oh, I had asked about TWA above, now I see it spelled out - be sure it's spelled out when the acronym is first used (above).</p><p> </p><p>I would make sure that either here, or ideally in the introduction above, a color-color diagram is defined and also its usefulness explained.</p><p> </p><blockquote>will be then to load this</blockquote><p> </p><p>I suggest remove "then".</p><p> </p><p>This section is really clearly explained with lots of detail! As the user I'm like, "Yes, I am following along!"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>